iocore/net/P_UnixNet.h

```diff
@@ -389,30 +400,171 @@ check_transient_accept_error(int res)
   }
 }
 
+static inline void
+net_activity(UnixNetVConnection *vc, EThread *thread)
+{
+  Debug("socket", "net_activity updating inactivity %" PRId64 ", NetVC=%p", vc->inactivity_timeout_in, vc);
+  (void)thread;
```

Comment:
Put back the `net_activity()` to `UnixNetVConnection.cc`.
I will merge our new Timeout System later and Reorganizing the `net_activity()`.